graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -211,6 +214,26 @@ public String aliasTarget(String alias) {
         return response.getAliases().isEmpty() ? null : response.getAliases().keysIt().next();
     }
 
+    public boolean createIndexTemplate() {
+        final Map<String, Object> template = indexMapping.messageTemplate(allIndicesAlias(), configuration.getAnalyzer());
+        final PutIndexTemplateRequest itr = c.admin().indices().preparePutTemplate(GRAYLOG_INTERNAL_TEMPLATE_NAME)
+                .setOrder(-99) // Make sure templates with "order: 0" are applied after our template!
```

Comment:
I chose -99 because I didn't want to use -1 because we might need some space in the future. :wink: 

I will change it to `Integer.MIN_VALUE`.